database/sqlite/sqlite_aclk_alert.c

```diff
@@ -521,25 +537,18 @@ void aclk_send_alarm_configuration(char *config_hash)
 
 int aclk_push_alert_config_event(char *node_id __maybe_unused, char *config_hash __maybe_unused)
 {
-    int rc = 0;
+    int rc;
```

Comment:
Not mine, but codacy complains here about no value. Likely if the ifdef below does not apply, we return a random value?